CHANGELOG.md

```diff
@@ -1,19 +1,20 @@
 # CHANGELOG
 
 
-## 0.12.0 (2023-xx-xx)
+## 0.12.0 (2023-01-13)
 
 * Add ``async with`` support to ``BaseCache``.
```

Comment:
FWIW it'd be nice to standardize on a consistent style natural to changelogs. Like describing what's changed since the last release in a non-imperative style... https://github.com/aio-libs/aiohttp/tree/master/CHANGES#alright-so-how-to-add-a-news-fragment